lib/nerve/reporter/zookeeper.rb

```diff
@@ -74,6 +73,8 @@ def stop()
     end
 
     def report_up()
+      log.info "nerve: adding zk node at #{@full_key}" if @full_key
```

Comment:
i think i would prefer these messages in `zk_create`, `zk_save` and `zk_delete` methods, as close to actual ZK operations as possible